### PR TITLE
simplify_with_basic_ops_pass UT

### PR DESCRIPTION
**Pull request by b3602sss**
### PR types
New features

### PR changes
OPs

### Describe
UT

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/ir/inference/test_simplify_with_basic_ops_pass_autoscan.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+            min_subgraph_size=0,
+            precision_mode=paddle_infer.PrecisionType.Float32,
+            use_static=False,
+            use_calib_mode=False)
+        yield config, ['scale', 'relu'], (1e-5, 1e-5)
+
+    def test(self):
+        self.run_and_statis(
+            quant=False,
+            max_examples=30,
```
Comment (jiweibo) [inline python/paddle/fluid/tests/unittests/ir/inference/test_simplify_with_basic_ops_pass_autoscan.py:148]: 30个case就可以覆盖所有情况吧

Comment (b3602sss) [reply python/paddle/fluid/tests/unittests/ir/inference/test_simplify_with_basic_ops_pass_autoscan.py:148]: 30个应该够了 pass简单 OP属性也少

**Review by jiweibo — DISMISSED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/ir/inference/CMakeLists.txt [hunk 2]
```diff
@@ -71,6 +71,7 @@ set_tests_properties(test_trt_matmul_quant_dequant PROPERTIES TIMEOUT 100)
 set_tests_properties(test_trt_conv3d_op PROPERTIES TIMEOUT 60)
 set_tests_properties(test_trt_conv3d_transpose_op PROPERTIES TIMEOUT 60)
 set_tests_properties(test_trt_nearest_interp_v2_op PROPERTIES TIMEOUT 30)
+set_tests_properties(test_simplify_with_basic_ops_pass_autoscan PROPERTIES TIMEOUT 120)
```
Comment (XieYunshen) [inline python/paddle/fluid/tests/unittests/ir/inference/CMakeLists.txt]: test_simplify_with_basic_ops_pass_autoscan设置成60S，Windows任务中的最长的执行时间是39S

**Review by XieYunshen — APPROVED** [review]
LGTM for `set_tests_properties(test_simplify_with_basic_ops_pass_autoscan PROPERTIES TIMEOUT 60)`

**Pull request merged**